Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -865,19 +870,14 @@ public final class ManifestLoader: ManifestLoaderProtocol {
 
     /// Returns the runtime path given the manifest version and path to libDir.
     private func runtimePath(for version: ToolsVersion) -> AbsolutePath {
-        // Bin dir will be set when developing swiftpm without building all of the runtimes.
-        if let binDir = self.toolchain.binDir {
-            return binDir
-        }
-        
-        // Otherwise we use the standard location of the manifest API in the toolchain, if it exists.
-        let manifestAPIDir = self.toolchain.libDir.appending(component: "ManifestAPI")
+        let manifestAPIDir = self.toolchain.swiftPMLibrariesLocation.manifestAPI
         if localFileSystem.exists(manifestAPIDir) {
             return manifestAPIDir
         }
-        
-        // Otherwise, fall back on the old location (this would indicate that we're using an old toolchain).
-        return self.toolchain.libDir.appending(version.runtimeSubpath)
+
+        // FIXME: how do we test this?
```

Comment:
@abertelrud do you know how I can test this? otherwise I was able to test all scenarios I was aware of